pyheif/reader.py

```diff
@@ -55,6 +55,34 @@ def close(self):
             del self._heif_handle
 
 
+class HeifContainer:
+    def __init__(self, primary_image, top_level_images):
+        self.primary_image = primary_image
+        self.top_level_images = top_level_images
```

Comment:
As I understood, `primary_image` is `UndecodedHeifFile`, while `top_level_images` is a list of `HeifTopLevelImage`. As primary_image could also have depth_image and auxiliary_images, doesn't it better to make `primary_image` `HeifTopLevelImage` also?

Reply:
Yes, this makes sense